Documentation/Signoffs/SensorSignoff.md

```diff
@@ -0,0 +1,94 @@
+
+## CONSTRAINTS ON THE SUBSYSTEM
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; The subsystem constraints are placed to ensure that it can fulfill its job in a satisfactory manner.  The constraints given below are structured to meet the qualifications of functionality, modularity, and sampling rate for the digital, analog voltage, and current sensors. Constraints on the subsystem are as follows:
+
+1. Shall have a selection of tested sensors to cover a range of pollutants, particulates, air composites, and other qualities.  Common qualities to measure are carbon monoxide, lead, nitrogen oxides, ozone, particulate matter, and sulfur dioxide.  Other qualities may be selected in the future, but these are definite qualities to look for.  These sensors should work independently from each other.
+
+2. Shall be modular. Specifically, the change of sensor should be seamless and handled by the board and software. The sensors will have multiple kinds of Input/Output (I/O) ports, so a list will be made that acknowledges all I/O ports of sensors to be approved and tested. The I/O ports that are shared among a group of sensors should be able to operate seamlessly if swapped out from the same type of I/O port for another sensor in the group using that I/O port.  Sensors that do not share this I/O port will have a port of its own grouping that has this same functionality within that group.
+
+3. Each sensor must be able to communicate with the microcontroller using a communicative language such as Serial Peripheral Interface (SPI), or Universal Asynchronous Receiver-Transmitter (UART). The microcontroller for this design will also be able to communicate with those same data transfer techniques. 
+
+4. Shall be able to operate with at least 3 sensors sampling independently of each other.  Sensors will take samples at set intervals and be read and stored on the device periodically.  Samples may be received and outputted one at a time.
+
+5. According to the Pacific Marine Environmental Laboratory [1], the speed for sample rates when receiving analog or digital signal input from the sensors should be sampled at a rate at approximately 2 Hz with a sample period of 30 seconds. Having sample rates that are too low causes data to be lost in the calibration process.
+
+6. The input voltages from the interface will provide either 3 or 5 V DC. Therefore, each sensor that will be used for the device shall have input voltages of either value. Appropriate current and power supply to the load will also have similar values for each sensor.
+
+7. For every sensor that is plugged onto the Air Quality Device initially, there may be a start up time for initial calibration. All sensors shall start data acquisition once the timer goes to 0, and the microcontroller signals the sensors to start.
+
+
+## BUILDABLES OF THE SUBSYSTEM
```

Comment:
Need circuit diagram, including pinouts, so the hardware can be assembled. Include all sensors in the diagram.

Reply:
I will create a schematic by tomorrow.